x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -1218,6 +1224,14 @@ function logActiveAndRecoveredAlerts<
         recoveredAlertIds
       )}`
     );
+
+    if (canSetRecoveryContext) {
+      for (const id of recoveredAlertIds) {
+        if (!recoveredAlerts[id].hasContext()) {
+          logger.debug(`No recovery context specified for recovered alert ${id}`);
```

Comment:
```suggestion
          logger.debug(`rule ${ruleLabel} has no recovery context specified for recovered alert ${id}`);
```

Reply:
Updated in https://github.com/elastic/kibana/pull/124972/commits/afe0c45743cd5a14e4bff3b5523846d02179058a